sdcm/nemesis.py

```diff
@@ -1379,8 +1379,12 @@ def _disrupt_kubernetes_then_replace_scylla_node(self, disruption_method):
 
         self.log.info('Mark %s (uid=%s) to be replaced', node, old_uid)
         node.wait_for_svc()
-        node.mark_to_be_replaced()
-        self._kubernetes_wait_till_node_up_after_been_recreated(node, old_uid=old_uid)
+        with DbEventsFilter(
+                db_event=DatabaseLogEvent.DATABASE_ERROR,
+                line="init - Startup failed.*seastar.*sleep_aborted"):
```

Comment:
I'm not sure `line` accepts regex.
Why not to use just `init - Startup failed: seastar::sleep_aborted` ?

If you use regex, need to add .* at the beginning and the end.

Reply:
Done